### PR TITLE
Update Trm sanctions check to use risk indicators

### PR DESCRIPTION
**Pull request by 1saf**
# Description

Uses risk indicators for TRM sanctions now.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [X] Other

## How should this be tested?

Test against a sanctioned wallet. Should work as before.

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [X] I have commented my code where relevant, particularly in hard-to-understand areas
- [X] If package-lock.json has changes, it was intentional.
- [X] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

**Comment by 1saf** [comment]
@timjrobinson @garethfuller changes made

File: src/services/web3/web3.plugin.ts [hunk 1]
```diff
@@ -195,16 +196,16 @@ export default {
         // need to store address to pre-load that connection
         if (account.value) {
           // fetch sanctioned status
-          let _isSanctioned = await isSanctionedAddress(account.value);
-          if (_isSanctioned === null) {
+          let _isBlocked = await isBlockedAddress(account.value);
+          if (_isBlocked === null) {
             // await disconnectWallet();
```
Comment (timjrobinson) [inline src/services/web3/web3.plugin.ts:201]: Can delete this now

File: src/constants/exploits.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ export type Exploits = Record<
   }
 >;
 
-export const SANCTIONS_ENDPOINT = 'https://api.balancer.fi/sanctions-check';
+export const WALLET_SCREEN_ENDPOINT = 'https://api.balancer.fi/sanctions-check';
```
Comment (timjrobinson) [inline src/constants/exploits.ts:11]: Could rename the endpoint to `/wallet-check` too.

**Review by timjrobinson — APPROVED** [review]

**Review by garethfuller — APPROVED** [review]
Thanks Ansaf, looks good to go except I agree with Tim that we should probably update the name of that endpoint since it's no longer accurate.

**Comment by timjrobinson** [comment]
Merged in as a hotfix in #2179

**Pull request closed without merging**